content/docs/gke/troubleshooting-gke.md

```diff
@@ -417,3 +417,53 @@ kubectl -n kubeflow create secret generic kubeflow-oauth \
        --from-literal=client_id=${CLIENT_ID} \
        --from-literal=client_secret=${CLIENT_SECRET}
 ```
+
+## Managed certificates can fail if service management API not enabled
+
+This section describes how to enable service management API to avoid managed certificates failure.
+
+To check your certificate: 
+1. Run the following command:
```

Comment:
The list formatting is broken here on the rendered page. See preview: https://deploy-preview-1664--competent-brattain-de2d6d.netlify.com/docs/gke/troubleshooting-gke/#managed-certificates-can-fail-if-service-management-api-not-enabled

I think adding an empty line before the first list item will fix this.

Reply:
Thanks for pointing this. 
I have added an empty line before the list. It helped to solve the issue.

Reply:
Thanks!